README.md

```diff
@@ -46,11 +46,11 @@ allow = [
 
 After that you can run:
 
-#### `$ cargo lint fmt`
```

Comment:
This isn't a type unless you change the name of the command, and if you do, lots of things used by other people will break.

Reply:
Not sure what you mean, when I install it, it is found/valid only as `cargo lints` not `lint`

Reply:
@appetrosyan I didn't change anything to invoke the tool as `lints` vs `lint`.

```bash
 > cargo lint
error: no such subcommand: `lint`

        Did you mean `lints`?   
 > cargo lints
cargo-lints-lints 0.1.0

USAGE:
    cargo-lints lints [OPTIONS] <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -f, --file <file>    Path to lints.toml file

SUBCOMMANDS:
    fmt     Formats lints.toml file
    help    Prints this message or the help of the given subcommand(s)
```